lib/jquery-ui-1-12.js

```diff
@@ -70,7 +74,8 @@ function get( data, key ) {
 
 function getDependencies( data ) {
 	var match = data.match( /define\((\ ?\[[\s\S]*?\]\ ?), factory \);/ );
-	return match && JSON.parse( match[ 1 ] ) || [];
+	// Remove line comments, if any
+	return match && JSON.parse( match[ 1 ].replace(/\/\/.+/g, '') ) || [];
```

Comment:
This deals with these two comments: https://github.com/jquery/jquery-ui/blob/a1d69208bad175a27c7b50c27fdc10001563cd4d/ui/widgets/button.js#L26-L27